src/cli/data.py

```diff
@@ -29,62 +34,6 @@ def _get_client(self):
 
             return client  # pragma: no cover
 
-    @cement.ex(
-        help="Merge two or more databases into a single table, integrating their cross-references, and save the table to a CSV file",
-        arguments=[
-            (
-                ["--databases"],
-                {
-                    "help": "List of IDs of the databases to merge",
-                    "action": "store",
-                    "nargs": "*",
-                    "required": True,
-                },
-            ),
-            (
-                ["--destination"],
-                {
-                    "type": str,
-                    "required": True,
-                    "metavar": "<destination>",
-                    "help": "Local directory path to save the table to",
-                },
-            ),
-            (
-                ["--include-files"],
-                {
-                    "action": "store_true",
-                    "help": "Whether to download the files in the databases [default: False]",
-                },
-            ),
-        ],
-    )
-    def merge_db(self):
-        """Merge databases and save as CSV"""
-
-        databases = self.app.pargs.databases
-        destination = self.app.pargs.destination
-        dfs = []
-        save_name = "-".join(databases)
-        for db in databases:
-            df = api.get_dataframe(
-                db,
-                client=self._get_client(),
-            )
-            dfs.append(df)
-        df = api.merge_databases(dfs)
-
-        # save to CSV
-        df.to_csv(os.path.join(destination, f"merged-{save_name}.csv"))
-
-        if not self.app.pargs.include_files:
-            return
-
-        for df in dfs:
-            files = df.attrs["file_ids"]
-            for file in files:
-                api.download_file(file, destination=destination)
-
```

Comment:
deleting because this functionality was removed

Reply:
Why did we remove this?

Reply:
Resolving so we can merge. We can restore this later.